packages/local_auth/local_auth_android/android/src/main/java/io/flutter/plugins/localauth/LocalAuthPlugin.java

```diff
@@ -269,13 +270,23 @@ private ArrayList<String> getAvailableBiometrics() {
     if (activity == null || activity.isFinishing()) {
       return biometrics;
     }
+
+    if (!hasBiometricHardware()) {
+      biometrics.add("undefined");
```

Comment:
This is the opposite of what the Dart code is expecting, so this doesn't fix my comment, and also breaks the reverse case.